sandbox/src/mocks/cancelReferral/responses/ExampleCancelledReferralWithAppointment.json

```diff
@@ -24,6 +24,12 @@
                             "reference": "#appointment"
```

Comment:
Not really part of this review, but I think the ID of the referral request here needs to be updated to match the UBRN parameter provided. I think it should be `000000070000` rather than `000000070001`.

Reply:
Yup, I noticed that and changed it in https://github.com/NHSDigital/e-referrals-service-api/pull/249/files#diff-316fe1a693c3ad09ac5e9284380fc744365ed7d79f0c025fd788fe8468b423d8